### PR TITLE
ollama: Update curl proxy.

### PR DESCRIPTION
**Pull request by feng-intel**
## Description
add noproxy in curl command.

## Issues
Need unset http_proxy and https_proxy, or curl failed.

## Type of change
doc

## Dependencies
No

## Tests
Pass

### CODE REVIEW

**Review by mkbhanda — APPROVED** [review]
I am puzzled why/when this proxy stuff changed. From India with proxy on my git clone commands stopped working! Is this related to the new auto VPN?

**Comment by mkbhanda** [comment]
@feng-intel  DCO missing! docker file path change.

**Review by lvliang-intel — APPROVED** [review]

File: comps/llms/text-generation/ollama/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ All of your local models are automatically served on localhost:11434. Run ollama
 Send an application/json request to the API endpoint of Ollama to interact.
 
 ```bash
-curl http://localhost:11434/api/generate -d '{
+curl --noproxy "*" http://localhost:11434/api/generate -d '{
```
Comment (ftian1) [inline comps/llms/text-generation/ollama/README.md:43]: shall we use env variable HTTP_PROXY, HTTPS_PROXY, NO_PROXY to control the behavior but not in curl cmd self?

if it doesn't work, another way may be asking user to edit .curlrc with their own proxy config. so the cmd line here is general for all cases.

**Review by Spycsh — APPROVED** [review]

File: comps/llms/text-generation/ollama/README.md (hunk 1 above)
Comment (mkbhanda) [reply comps/llms/text-generation/ollama/README.md:43]: Personally I like the idea of editting .curlrc file. Even for GitHub commands like pull etc, I needed different settings depending on where I worked from. So let us just provide documentation, "assuming you have set your .curlrc to reflect proxy settings that you may need in your environment. Example settings are: .."

**Pull request merged**